From 2845d329ab30d3d9ef32bc52b46933c9d92da08c Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Fri, 5 Jan 2024 03:07:58 +0000
Subject: [PATCH 64/64] WIP: arm64: dts: meson: fixup Minix U9-H wifi

I think the 'drop compatible' change conflicted so remove this too.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxm-minix-neo-u9h.dts | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxm-minix-neo-u9h.dts b/arch/arm64/boot/dts/amlogic/meson-gxm-minix-neo-u9h.dts
index c3fb523fd18e..bed70c5c2d9c 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxm-minix-neo-u9h.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-gxm-minix-neo-u9h.dts
@@ -99,13 +99,6 @@ rtc: rtc@51 {
 	};
 };
 
-&sd_emmc_a {
-	brcmf: wifi@1 {
-		reg = <1>;
-		compatible = "brcm,bcm4329-fmac";
-	};
-};
-
 &sd_emmc_b {
 	/* experimental */
 	sd-uhs-sdr12;
-- 
2.34.1

